storage/common.go

```diff
@@ -73,6 +73,8 @@ type Queue interface {
 	Enqueue(Value) error
 	//Dequeue removes a FIFO element
 	Dequeue() (Value, error)
+	// //Peekqueue returns the first element
```

Comment:
Is this comment correct? Not all elements?

Reply:
Let's also add a limit. so `Peek(limit int)` with defaulting to the storage limit, otherwise we might end up stalling the queue for too much.

Reply:
updated the comments.

Added the limit condition of upto 1000. ( I just picked that number )

Reply:
https://github.com/storj/storj/blob/master/storage/common.go#L44

Reply:
Done...